velox/exec/fuzzer/AggregationFuzzer.cpp

```diff
@@ -389,6 +423,10 @@ void AggregationFuzzer::go() {
         }
 
         auto input = generateInputData(argNames, argTypes, signature);
+        std::shared_ptr<ResultVerifier> customVerifier;
+        if (customVerification) {
```

Comment:
It is only used to sort the result of agg like `array_agg` to avoid the error as the follows,
```C++
-- Aggregation[SINGLE [g0] a0 := array_agg(ROW["c0"]) distinct] -> g0:INTEGER, a0:ARRAY<INTEGER>
  -- Values[1000 rows in 10 vectors] -> g0:INTEGER, c0:INTEGER
I20240111 22:09:17.256809 2330690 Task.cpp:1082] All drivers (1) finished for task test_cursor 21 after running for 4 ms.
I20240111 22:09:17.256819 2330690 Task.cpp:1766] Terminating task test_cursor 21 with state Finished after running for 4 ms.
/Users/macduan/work/native/velox-8/velox/exec/tests/utils/QueryAssertions.cpp:1104: Failure
Failed
Expected 295, got 295
21 extra rows, 21 missing rows
10 of extra rows:
        ...
        134009693 | [1589920836,1168214174]
        171478292 | [null,1392033972]
        173011053 | [470587781,2061038386,1520225581]
        334559877 | [952534536,1650626939]
        ...
10 of missing rows:
        ...
        134009693 | [1168214174,1589920836]
        171478292 | [1392033972,null]
        173011053 | [2061038386,1520225581,470587781]
        334559877 | [1650626939,952534536]
        ...
```

Reply:
There are different types of custom verifiers. Not all of them are aware of distinct aggregations. We need to either (1) update verifier API to require verifiers to respect distinct aggs or (2) update verifier API to allow verifier to declare whether it supports distinct agg or not and use that information to disable custom verification.

Reply:
I've added a comment to explain why we exclude "approx_" and updated the PR description to clarify as well per our discussion.